### ISSUE TITLE
As user, I cannot login when passbolt is installed in subdir

### ISSUE THREAD

**Issue by cmouse** [passbolt/passbolt_api#229]
* Passbolt Version: 2.0.0-rc2
* Platform and Target: 
-- Operating system: Ubuntu 16.04.3
-- PHP: 7.0.25-0ubuntu0.16.04.1
-- Web server: 2.4.18-2ubuntu3.5
-- Database server: 10.0.33-0ubuntu0.16.04.1

passbolt.php
```.php
return [
  'App' => [
        'fullBaseUrl' => 'https://admin/passbolt',
    ],
  'Datasources' => [
        'default' => [
            'host' => 'localhost',
            'username' => 'passbolt',
            'password' => '<redacted>',
            'database' => 'passbolt',
        ],
    ],
    'EmailTransport' => [
        'default' => [
            'host' => 'mail',
            'port' => 25,
            'username' => '',
            'password' => '',
            'tls' => true,
        ],
    ],
    'Email' => [
        'default' => [
            'from' => ['passbolt@domain' => 'Passbolt'],
            'charset' => 'utf-8',
            'headerCharset' => 'utf-8',
        ],
    ],

    'passbolt' => [
        'gpg' => [
            'keyring' => getenv("HOME") . DS . '.gnupg',
            'serverKey' => [
                'fingerprint' => '<redacted>',
                'public' => CONFIG . 'gpg' . DS . 'public.asc',
                'private' => CONFIG . 'gpg' . DS . 'private.asc',
            ],
        ],
    ],
];
```

### What you did
I installed 2.0.0-rc2
 - compose install
 - bin/cake passbolt install

Then I registered my first user, and activated that user.

### What happened
When logging in, it attempts to load `https://admin/passbolt/passbolt/users/16ca5493-f055-44a7-b038-6f37fa08ea1a.json?async=false`

### What you expected to happen
When logging in, it should be loading
`https://admin/passbolt/users/16ca5493-f055-44a7-b038-6f37fa08ea1a.json?async=false`

**Comment by stripthis** [comment]
Hi @cmouse,

Thanks for the bug report. I could reproduce the issue, will be publishing a fix for this with the next release.

**Comment by cedricalfonsi** [comment]
Fixed with v2.0.0

**Comment by cmouse** [comment]
Thank you =)
